store/src/java/com/zimbra/cs/service/mail/CreateDataSource.java

```diff
@@ -252,6 +254,21 @@ static void validateFolderId(Account account, Mailbox mbox, Element eDataSource,
             }
         }
     }
+    /**
+     * Confirms that the zimbraDataSourceEmailAddress attribute is unique
+     * 
+     */
+    default static void validateEmailAddr(Account account, Element eDataSource)
```

Comment:
validateEmailAddr() sounds more like email address validation than data source validation, may be changed to validateEmailDataSource()

Reply:
fixed